source/dpq/attributes.d

```diff
@@ -221,11 +218,18 @@ template relationName(alias R)
 {
    static if (hasUDA!(R, RelationAttribute))
    {
-      enum rName = getUDAs!(R, RelationAttribute)[0].name;
-      static if (rName.length == 0)
+      // check if @relation is a type
+      static if (is(getUDAs!(R, RelationAttribute)[0]))
```

Comment:
This is the cunning way to check if our `@relation` UDA is a type or is a value.